src/modules/AutoinstSoftware.rb

```diff
@@ -151,12 +151,14 @@ def Import(settings)
         to:   "list <string>"
       )
 
-      PackageAI.toinstall = settings.fetch("packages", [])
+      to_install = settings.fetch("packages", [])
+      PackagesProposal.AddResolvables("autoyast", :package, to_install) unless to_install.empty?
```

Comment:
Is it AddResolvables or SetResolvables? Is expected to have more there?

Reply:
I preferred to be conservative and use `AddResolvables`, just in case for some reason we add more packages from a different place. In that case, the selection would get overriden.